job/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -656,7 +657,7 @@ public int hashCode() {
             }
             node.addAction(new TimingAction());
 
-            logNodeMessage(node, "Running: " + node.getDisplayName());
+            logNodeMessage(node);
```

Comment:
Yeah, I see the "Running:" prefix at the beginning of each metadata line as a way to easily identify metadata logs, but it is redundant now since we have the "[Workflow]" prefix, and it is quite obvious that a step is "Running" when you see something like `[Workflow] node {` or `[Workflow] echo`.

The plain text version would be something like this (where is still easy to identify the metadata log lines IMO):

```
Started by user anonymous
[Workflow] Allocate node : Start
Running on master in /home/amuniz/workspace-cloudbees/workflow-plugin/aggregator/work/jobs/test/workspace
[Workflow] node {
[Workflow] echo
Hi!
[Workflow] git
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/amuniz/maven-helloworld.git # timeout=10
Fetching upstream changes from https://github.com/amuniz/maven-helloworld.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/amuniz/maven-helloworld.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision 197d621fea6a797cbb4a158df573580a83fd8b4f (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 197d621fea6a797cbb4a158df573580a83fd8b4f
 > git rev-list 197d621fea6a797cbb4a158df573580a83fd8b4f # timeout=10
[Workflow] stage: After Git Clone
Entering stage After Git Clone
Proceeding
[Workflow] echo
Bye!
[Workflow] sh
[workspace] Running shell script
 mvn clean install
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building example 1.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ example ---
...
```